logging/jsonlog/example_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jsonlog_test
+
+import (
+	"io"
+	"net/http"
+	"os"
+
+	"cloud.google.com/go/logging/jsonlog"
+)
+
+func ExampleNewLogger() {
+	l, err := jsonlog.NewLogger("projects/PROJECT_ID")
+	if err != nil {
+		// TODO: handle error.
+	}
+	l.Infof("Hello World!")
+}
+
+func ExampleNewLoggerFromRequest() {
+	// Create a Logger with additional information pulled from the current
+	// request context.
+	var req *http.Request
+	l, err := jsonlog.NewLoggerFromRequest("projects/PROJECT_ID", req)
```

Comment:
Neat to have common requests on a logger level! Can we also continue to allow users to just log a Httprequest obj without having to create a special request-logger?

Otherwise, for consistency with common labels, can we impl this as: 
`jsonlog.NewLogger()`
`l.WithRequest`